File: QE-selenium/python/Activity6.py
import time
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as dirver:
    dirver.get("https://training-support.net/webelements/dynamic-controls")
    print(dirver.title)
    print(dirver.find_element(By.XPATH,"//input[@id='checkbox']").is_selected())
    dirver.find_element(By.XPATH,"//input[@id='checkbox']").click()
    print(dirver.find_element(By.XPATH,"//input[@id='checkbox']").is_selected())
    dirver.quit()

File: QE-selenium/java/Activity6.java
package demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity6 {

    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net/webelements/dynamic-controls");

        System.out.println(driver.getTitle());

        System.out.println(driver.findElement(By.xpath("//input[@id='checkbox']")).isSelected());

        driver.findElement(By.xpath("//input[@id='checkbox']")).click();

        System.out.println(driver.findElement(By.xpath("//input[@id='checkbox']")).isSelected());

        driver.quit();
    }
}
